protocols/exmp-sampling.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+permalink: /protocols/exmp/sampling/
+color: "#593001"        #poop brown
+---
+
+
+#                   EXMP Fecal Sampling Instructions
+
+
+Please read through all of these instructions *BEFORE* you begin collecting a fecal sample.
+
+Trillions of harmless or beneficial microorganisms (tiny living things such as bacteria) live on and within our bodies. During and after birth, our bodies begin to accumulate this diverse community of microorganisms. The composition of this community differs between people and the scientific community is still exploring why this is. There is evidence that our microbial communities are influenced by our parents, our physical environment, what we eat and potentially even our genes. Our resident microbes occupy many body habitats, including the skin and mucosal surfaces, and the gastrointestinal tract.
+
+As part of this study, you will be asked to donate fecal samples of your microorganisms that will be collected on sterile cotton tipped swabs (i.e., “Q­tips”).  You will perform the sampling. Ziploc bags containing swabs will be distributed at the initial meeting (2 kits) and after that weekly from the drop off location.
+
+Samples should be taken each Sunday and Thursday following the instructions detailed below.
+
+  *	Please note: there are no back-up swabs.
+  *	If you accidentally drop the swab, or touch it to anything other than the specific sampling site, please skip that sample. The sample can be simply thrown away.
+
+You will fill out one Per Sample Event Questionnaire for every sample collected.
+
+#                 Step by step directions for fecal sampling
+
+
+You will be provided kits to collect fecal swab samples, these should be collected on Sunday and Thursday (or as close to these days as possible). Each time you collect a sample, follow these instructions. Collect each of these 2 weekly samples, one at a time, on Sunday and Thursday.
+
+  1.	From the zip­lock bag provided by the researchers, take out a labeled tube.  
+  2.	Take the cotton swab out of the labeled tube, handling the swab only by the red plastic handle.  
+  3.	Sampling protocol
+   *	You are asked to donate microorganisms from the intestinal microhabitat by collecting a fecal smear from used bathroom tissue by taking a swab of the your used tissue.
+   *	To do this, rub the swab over the used portion of the tissue until there is visible fecal material on the swab (i.e. the swab is brown).
+  4.	Return the sample swab to the labeled tube immediately after sampling and close the cap securely.
+  5.	Return the swab to the Ziploc bag.
+  6.	Be sure to wash your hands with soap and water after sampling to avoid contamination.  
```

Comment:
Does this all render correctly? This list changes here from two spaces between the number and the text to one space.